### DISCUSSION TITLE
Smoothing algorithm.

### DISCUSSION THREAD

**Discussion by sergey900553** [netless-io/flat#1733] [Q&A] [answered]
What algorithm is used when drawing with a pencil? Can you please show the implementation file?
![smoothing](https://user-images.githubusercontent.com/42143402/196419161-76944a70-629d-405c-b0a7-2a091b075cdc.gif)

**Comment by Leooeloel** [comment, marked as answer]
We used Agora Interactive Whiteboard SDK

https://www.agora.io/en/products/interactive-whiteboard
